sky/serve/serve_utils.py

```diff
@@ -758,7 +785,8 @@ def format_service_table(service_records: List[Dict[str, Any]],
         return 'No existing services.'
 
     service_columns = [
-        'NAME', 'VERSION', 'UPTIME', 'STATUS', 'REPLICAS', 'ENDPOINT'
+        'NAME', 'VERSION', 'UPTIME', 'STATUS', 'REPLICAS', 'ENDPOINT',
+        'TLS_ENCRYPTED'
```

Comment:
Do we need to add this column, considering the endpoint already indicate that? We can avoid this column for simplicity.